src/framework/preferences/css/stylus/utils/Helpers.styl

```diff
@@ -21,3 +21,23 @@ linear-gradient() {
         s('linear-gradient(%s)', content);
     }
 }
+
+
+// Mixin for common styles of the "thumb" (slider control)
+// used by both the jQuery and native HTML sliders
+slider-thumb-common() {
+    height: 1.5em;
+    width: 1.5em;
+    background-color: #fff;
+    border: 1px solid #b3b3b3;
+    border-radius: 2em;
+    box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
```

Comment:
Edit: I see now that this styling was written elsewhere and moved to this function. I guess if it's been working we should leave it. We should make sure this gets fixed in the redesign.